File: CS50x 2021/Week 1/3- PSet 1 Mario (Less)/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Notification
    printf("Welcome, Please enter a number from one to eight to build.\n");

    int h;
    do
    {
        //Getting height.
        h = get_int("Height: ");

    }
    while (1 > h || h > 8);

//Build.
    for (int l = 0; l < h; l++)
    {
        for (int s = h - l; s > 1; s--)
        {
            printf(" ");
        }
        for (int hash = 0; hash < l + 1; hash++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: CS50x 2021/Week 6/3- PSet 6 Sentimental ' Mario (Less)/mario.py
from cs50 import get_int

# Notification
print("Welcome, Please enter a number from one to eight to build.")
# Getting height.
h = 0
while 1 > h or h > 8:
    h = get_int("Height: ")
# Build.
for i in range(h):
    print((h - 1 - i) * " ", end="")
    print((i + 1) * "#")